stdlib/src/unicodedata.rs

```diff
@@ -113,6 +121,17 @@ mod unicodedata {
             Ok(bidi.to_owned())
         }
 
+        #[pymethod]
```

Comment:
```suggestion
        /// NOTE: This function uses 10.0.0 database instead of 3.2.0
        #[pymethod]
```

Maybe this is enough for the version mismatch for now.

Reply:
I see.
Current `unic` functions use v10.0.0, and in this PR we can introduce `ucd` crate for `unicodedata.east_asian_width` using v9.0.0.
Then we just add the above comment and leave `unicodedata.ucd_3_2_0.east_asian_width` as is for now.
Let me remove the example commit for `icu` and update the PR.

Reply:
PR is updated as in commit https://github.com/RustPython/RustPython/commit/7661fab743fb06baab3c86f60d6dedbd07b6b72b .